### ISSUE TITLE
(Bug can only be found when using controller) If you start sprinting, then move joystick slightly to the left, the camera will struggle to keep up with Niko.

### ISSUE THREAD

**Issue by ProbablyWorth** [GTAmodding/GTAIV-Issues-List#139]
https://youtu.be/IH8qEURZvPU
Just look at the surroundings, you can see the camera slowing down, speeding up, slowing down..... and this happens forever when you sprint slightly to the left. This can't be done on keyboard because Niko can only do a hard left.

I'm not touching the right joystick. I'm using my right thumb to mash A in order to sprint

**Comment by Tomasak** [comment]
Did you  try checking if this happens at 30 FPS? Or it only happens when turning left, but turning right is correct?

**Comment by ProbablyWorth** [comment]
> Did you try checking if this happens at 30 FPS? Or it only happens when turning left, but turning right is correct?

Yes, nothing happens when going right. Well test 30 FPS and come back. This bug occurs even when locked at 60 FPS and a bunch of fix mods

**Comment by ProbablyWorth** [comment]
> Did you try checking if this happens at 30 FPS? Or it only happens when turning left, but turning right is correct?

Still happens at 30 FPS. If you need more info, im using Steam CE, downgraded to 1.0.8.0

**Comment by Tomasak** [comment]
Interesting, to me it doesnt happen in TBOGT, but does in IV.
edit: tlad also seems fine.

**Comment by Tomasak** [comment]
PC move_player.wad is broken, tlad and tbogt uses their own and dont suffer from same issue

**Comment by ProbablyWorth** [comment]
> PC move_player.wad is broken, tlad and tbogt uses their own and dont suffer from same issue
> 
> 

Looking forward to a fix since the file causing the problem is found. Thx👍

**Comment by Tomasak** [comment]
[sprint.zip](https://github.com/GTAmodding/GTAIV-Issues-List/files/15368896/sprint.zip)
Parallellines tried to fix it, check this. Contains both PC and console

**Comment by ProbablyWorth** [comment]
> [sprint.zip](https://github.com/GTAmodding/GTAIV-Issues-List/files/15368896/sprint.zip) Parallellines tried to fix it, check this. Contains both PC and console

Tested both the PC and Console and the camera is still buggy, or maybe I didn't see if the files were actually replaced in OpenIV

**Comment by Tomasak** [comment]
if you have console visuals installed, replace it in update\ConsoleVisuals\IV\ConsolePlayerAnims.img
If you dont, you can throw the file into update\GTAIV.EFLC.FusionFix\IV\GTAIV.FusionFix.img

**Comment by ProbablyWorth** [comment]
> if you have console visuals installed, replace it in update\ConsoleVisuals\IV\ConsolePlayerAnims.img If you dont, you can throw the file into update\GTAIV.EFLC.FusionFix\IV\GTAIV.FusionFix.img

Wow, it works. However, the camera lag is still there but extremely small. And it's still very obvious when sprinting right with a two-handed weapon (sniper rifle, assault rifle, etc.)

**Comment by Tomasak** [comment]
[fixed anims.zip](https://github.com/GTAmodding/GTAIV-Issues-List/files/15455849/fixed.anims.zip)
try this one, it fixes more now

**Comment by ProbablyWorth** [comment]
> [fixed anims.zip](https://github.com/GTAmodding/GTAIV-Issues-List/files/15455849/fixed.anims.zip) try this one, it fixes more now

Works perfectly. All animations look great. Thanks!

**Comment by Tomasak** [comment]
https://github.com/ThirteenAG/GTAIV.EFLC.FusionFix/pull/490

**Comment by RecklessGlue540** [comment]
This is definitely nice to see fixed : )
